tests/cc55/input.dat

```diff
@@ -96,11 +96,11 @@ for transition, data in test_data.items(): # TEST
     for property, value in data.items(): # TEST
         # Change these numbers to 6 ASAP # TEST
         if "EINSTEIN" not in property: # TEST
-            compare_values(value, wfn.variable(f"CC {transition} {property}"), 6, f"CC {transition} {property}") # TEST
-            compare_values(value, wfn.variable(f"CCSD {transition} {property}"), 6, f"CCSD {transition} {property}") # TEST
+            compare_values(value, wfn.variable(f"CC {transition} {property}"), 4, f"CC {transition} {property}") # TEST
```

Comment:
any clue what the above `# Change these numbers to 6 ASAP` is about, here? it's suspect if these oscillator strengths aren't matching to at least 1e-6.

Reply:
It's unjustifiable to check these quantities to 1e-6 when cc [refuses to enforce lambda convergence any tighter than 1e-4](https://github.com/psi4/psi4/blob/master/psi4/driver/procrouting/proc.py#L3161-L3163). I have plans to fix this. [See](https://github.com/psi4/psi4/issues/2519) [`cclambda`'s](https://github.com/psi4/psi4/issues/2520) [sin](https://github.com/psi4/psi4/issues/25221) [list](https://github.com/psi4/psi4/issues/2522).

But that scale of refactoring is outside the scope of this review.

Reply:
wow. that is highly irregular, and definitely a problem. I'll be looking forward to these refactors. thanks!